proto/penumbra/penumbra/core/stake/v1alpha1/stake.proto

```diff
@@ -90,7 +90,14 @@ message BondingState {
     BONDING_STATE_ENUM_UNBONDED = 3;
   }
   BondingStateEnum state = 1;
-  optional uint64 unbonding_epoch = 2;
+
+  message UnbondingEpoch {
+    // The epoch in which stake becomes unbonded.
+    uint64 epoch = 1;
+  }
+
+  // Present if needed to specify unbonding epoch.
+  UnbondingEpoch unbonding_epoch = 2;
```

Comment:
We could also just have `uint64 unbonding_epoch = 2`, since `0` is not a valid unbonding epoch.